### DISCUSSION TITLE
Change the order of my issues in milestones

### DISCUSSION THREAD

**Discussion by igor2x** [dbeaver/dbeaver#18267] [DBeaver Development]
Three of my bug reports have been set as milestone:
1. Milestone 22.3.1: https://github.com/dbeaver/dbeaver/issues/17627
2. Milestone 22.3.2: https://github.com/dbeaver/dbeaver/issues/17704
3. Milestone 22.3.4 https://github.com/dbeaver/dbeaver/issues/17698

For me issue 3 is ten times more important then 1 and 2, but unfortunately issue 3 has been assigned at the latest milestone.

If I have any kind of influence, can issue 1 and 3 please be swapped in milestones.

**Comment by ShadelessFox** [comment]
I've assigned priorities for each issue.

The third one is now major, meaning it will likely reach one of the nearest milestones.

**Comment by igor2x** [comment]
Issue https://github.com/dbeaver/dbeaver/issues/17698  is still not in any milestone. Is it possible to assign it to a milestone, please.

**Reply by ShadelessFox** [reply]
👌🏻
